### PR TITLE
rough draft of script v0

### PR DESCRIPTION
**Pull request by britttttk**
Relates to #13.

### CODE REVIEW

File: Animation-Script-Ideas-1-4-21.md [hunk 1]
```diff
@@ -1 +1,28 @@
-Work In Progress coming back to start script.
+Scene 1: Apartment
+Characters: Father & Daughter
+Character Animation Bubbles: "People have bread. We make bread. Why does it have to be this hard?"
+
+Narration: 
+When two parties transact between one another, it's called a payment. 
+As simple as that sounds, these days it can be pretty complicated.
+Third parties and middleman are often times, far too involved in something simple like a baker selling his bread to people who need food.
```
Comment (pavlenex) [inline Animation-Script-Ideas-1-4-21.md:8]: Maybe mention fees (people relate to fees the most)

File: Animation-Script-Ideas-1-4-21.md [hunk 2]
```diff
@@ -1 +1,28 @@
...
+
+Scene 2: Market
+Characters: Father & Daughter - closing old shop
+Character Animation Bubbles: "It's just different. Times are changing. Everything will be fine."
+Narration:
+Opening a store or online business has become daunting for some.
+Others open a business and spent lots of time buried in paperwork, unnecessesry forms and fees for simply doing what they love.
```
Comment (pavlenex) [inline Animation-Script-Ideas-1-4-21.md:15]: Love this.

File: Animation-Script-Ideas-1-4-21.md [hunk 3]
```diff
@@ -1 +1,28 @@
...
+
+Scene 3: Market
+Characters: Father & Daughter - opening new shop
+Character Animation Bubbles: "People are in and out so quicky. Payments are already in our Bitcoin wallet"
+Narration:
+BTCPay is an entire payments platform, helping people achieve financial self-soverignty.
+... BTCPay description ...
```
Comment (pavlenex) [inline Animation-Script-Ideas-1-4-21.md:22]: Just make sure to mention it's for **Bitcoin** payments.

**Review by pavlenex — COMMENTED** [review]
Love it. Great job. Just a few brain dumps.

File: Animation-Script-Ideas-1-4-21.md (hunk 3 above)
Comment (pavlenex) [reply Animation-Script-Ideas-1-4-21.md:22]: self-soverignty. may be financial freedom/independence, I am just thinking self-soverignty may be hard to translate

Comment (Zaxounette) [reply Animation-Script-Ideas-1-4-21.md:22]: Financial freedom will be more easily understood, translatable and conveyable than self-sovereignty in French for sure.
I expect it be the same in a number of other languages.

**Comment by Zaxounette** [comment]
Merging this,

I need it to continu working on the script.

Will link to Pav's brain dump in the next PR.

**Pull request merged**